build-system/common/update-design-review-issues.js

```diff
@@ -308,61 +317,45 @@ function unpinGithubIssue(token, repo, number) {
 
 /**
  * @param {Date} date
- * @param {number} dayOfWeek
- * @param {number=} weeks
+ * @param {number=} days
  * @return {Date}
  */
-function getNextDayOfWeek(date, dayOfWeek, weeks = 1) {
+function addDays(date, days = 1) {
   const resultDate = new Date(date.getTime());
-  resultDate.setDate(
-    resultDate.getDate() +
-      (weeks - 1) * 7 +
-      ((7 + dayOfWeek - date.getDay()) % 7)
-  );
+  resultDate.setDate(resultDate.getDate() + days);
   return resultDate;
 }
 
 /**
- * @param {Date} date
- * @param {string} startYyyyMmDd
- * @return {Array<string>}
+ * @param {Date} nextDay
+ * @param {Date} start
+ * @return {RotationItemDef}
  */
-function getRotation(date, startYyyyMmDd) {
-  const [year, month, day] = startYyyyMmDd
-    .split('-')
-    .map((n) => parseInt(n, 10));
-  const start = new Date(year, month - 1, day);
-  const dateBeginningOfDay = new Date(
-    date.getFullYear(),
-    date.getMonth(),
-    date.getDate()
-  );
-  const weeks = Math.round(
-    // @ts-ignore date calc
-    (dateBeginningOfDay - start) / (7 * 24 * 60 * 60 * 1000)
-  );
+function getRotation(nextDay, start) {
+  // @ts-ignore date calc
+  const delta = nextDay - start;
+  const weeks = Math.round(delta / (7 * 24 * 60 * 60 * 1000));
   return timeRotationUtc[weeks % timeRotationUtc.length];
 }
 
 const timeZ = (yyyy, mm, dd, hours, minutes) =>
-  `${yyyy + mm + dd}T${leadingZero(hours) + leadingZero(minutes)}Z`;
+  `${yyyy + mm + dd}T${leadingZero(hours) + leadingZero(minutes)}00Z`;
```

Comment:
Trying to get this in as a freebie. `Z` times in the Google Calendar link require seconds. When they're absent, the times are offset 5 mins in the future (for some reason). This fixes it.